src/styles/blocks/card.scss

```diff
@@ -0,0 +1,156 @@
+}
+
+@media (min-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media (min-width: 768px) {
+  .cards {
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards {
+    grid-template-columns: repeat(4, $card-width);
+  }
+}
+
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
+  transition: transform 300ms ease;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__product-details {
+    width: $card-content-width-adjustment;
+    margin-inline: $padding-inline;
+    margin-bottom: $margin-bottom;
+
+    &-title {
+      height: 36px;
+      font-size: $font-size-medium;
+      font-weight: $font-weight-medium;
+      line-height: 1.5;
+      text-align: left;
+      color: $secondary-color;
+      margin-top: 0;
+      margin-bottom: 4px;
+    }
+
+    &-description {
+      margin: 0;
+      font-size: $font-size-small;
+      line-height: 14px;
+      text-align: left;
+      color: $text-color;
+    }
+  }
+
+  &__rating {
+    box-sizing: border-box;
+    width: $card-content-width-adjustment;
+    height: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-inline: $padding-inline;
+    margin-bottom: 24px;
+
+    &-reviews {
+      font-size: $font-size-small;
+      line-height: 14px;
+      text-align: right;
+      color: $secondary-color;
+    }
+  }
+
+  &__price {
+    box-sizing: border-box;
+    width: $card-content-width-adjustment;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-inline: $padding-inline;
+    margin-bottom: $margin-bottom;
+
+    &-title {
+      margin: 0;
+      font-size: $font-size-medium;
+      line-height: 18px;
+      text-align: left;
+      color: $text-color;
+    }
+
+    &-cost {
+      margin: 0;
+      font-size: $font-size-large;
+      font-weight: $font-weight-bold;
+      line-height: 18px;
+      text-align: right;
+      color: $secondary-color;
+    }
+  }
+
+  &__button-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    border-radius: $border-radius;
+    background-color: $primary-color;
+    margin-inline: $padding-inline;
+    margin-bottom: $margin-bottom;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: $font-weight-bold;
+    line-height: 16px;
+    text-align: center;
+    color: $white-color;
+
+    &:hover {
+      box-sizing: border-box;
+      background-color: $white-color;
+      border: 1px solid $primary-color;
+      color: $primary-color;
+    }
+  }
+}
+
+.product-card:hover {
+  transform: scale(1.2);
+}
```

Comment:
Consider using a more subtle scaling factor for the hover effect to prevent content around from moving too much, something like '1.05' or '1.1' could be more user-friendly.